cmd/provenanced/cmd/addresses.go

```diff
@@ -3,16 +3,35 @@ package cmd
 import (
 	"fmt"
 
+	"github.com/cosmos/cosmos-sdk/client"
 	"github.com/google/uuid"
 	"github.com/spf13/cobra"
 
 	"github.com/provenance-io/provenance/x/metadata/types"
 )
 
+// GetQueryCmd is the top-level command for name CLI queries.
+func AddMetaAddressCmd() *cobra.Command {
+	queryCmd := &cobra.Command{
+		Use:                        "metaaddress",
+		Short:                      "Decode/Encode Metaaddresses commands",
```

Comment:
`Aliases: []string{"ma"}` might be nice here. `"mda"` might also be a good one.

Reply:
Just added the `ma` command.  Added `e` and `d` to encoder and decoder.